### DISCUSSION TITLE
How can I edit only part of template

### DISCUSSION THREAD

**Discussion by henzigo** [GrapesJS/grapesjs#4100] [Q&A]
Hello,

I want to edit only part of template in editor. I got some help, how I'm able to disable some parts of my template, but this is not exactly what I need. I tried to figure out how can I invert logic of code pasted bellow, but without any success.

> If you need to disable parts of your template, this is probably what you're looking for: https://jsfiddle.net/artur_arseniev/w6mvq4bx/

I will try to describe my situation:

I have using GrapesJS for editing blog articles. I want to make the use more enjoyable so before I initialize GrapesJS I download `template` of my frontend (This template is created dynamically, so it can be changed everytime I'm loading editor) and show this template in editor. Unfortunately I need to allow editing, dragging, dropping etc. only one nested div, so I can't use example above. (The best way would be to invert logic and allow editing only for div with specific class). 

Here is how my template page looks like on frontend
<img width="687" alt="Snímek obrazovky 2022-01-21 v 16 31 13" src="https://user-images.githubusercontent.com/1482966/150554768-a2b4859a-1b2c-4ed9-84a2-91ba9a200ea6.png">

After I'm done with styling article, I need save to my database only content of that nested div, so I can load contend via API on frontend.

Here is how my editor with loaded template looks like:
<img width="1458" alt="Snímek obrazovky 2022-01-21 v 16 17 53" src="https://user-images.githubusercontent.com/1482966/150552111-e632f6db-d2ab-456f-a451-853e87ce85a1.png">

Is there any way, how can I achieve my requirements at this time? Maybe in future, It would be nice to have some selector in configuration, that will do what I need. What do you think? 

Thank you for your advice ❤️

**Comment by artf** [comment]
Sorry @henzigo replied to you already on your original issue https://github.com/artf/grapesjs/issues/4083
I'll post there the demo example once the new release, with fixes, is published.
